judo-runtime-core-jsl-itest/common/src/main/java/hu/blackbelt/judo/runtime/core/jsl/fixture/JudoHsqldbModulesTest.java

```diff
@@ -0,0 +1,108 @@
+package hu.blackbelt.judo.runtime.core.jsl.fixture;
+
+/*-
+ * #%L
+ * JUDO Runtime Core :: Parent
+ * %%
+ * Copyright (C) 2018 - 2022 BlackBelt Technology
+ * %%
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License, v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is
+ * available at https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ * #L%
+ */
+
+import com.google.inject.Singleton;
+import com.google.inject.name.Names;
+import com.google.inject.util.Providers;
+import hu.blackbelt.judo.dispatcher.api.Sequence;
+import hu.blackbelt.judo.runtime.core.bootstrap.JudoModule;
+import hu.blackbelt.judo.runtime.core.bootstrap.dao.rdbms.hsqldb.*;
+import hu.blackbelt.judo.runtime.core.dao.rdbms.Dialect;
+import hu.blackbelt.judo.runtime.core.dao.rdbms.RdbmsInit;
+import hu.blackbelt.judo.runtime.core.dao.rdbms.RdbmsParameterMapper;
+import hu.blackbelt.judo.runtime.core.dao.rdbms.hsqldb.HsqldbDialect;
+import hu.blackbelt.judo.runtime.core.dao.rdbms.query.mappers.MapperFactory;
+import lombok.Builder;
+import org.hsqldb.server.Server;
+
+import javax.sql.DataSource;
+import java.io.File;
+
+import static hu.blackbelt.judo.runtime.core.bootstrap.dao.rdbms.hsqldb.HsqldbServerProvider.*;
+
+public class JudoHsqldbModulesTest extends JudoModule {
+
+    private Boolean runServer = false;
+    private String databaseName = "judo";
+    private File databasePath = new File(".", "judo.db");
+    private Integer port = 31001;
+
+
+    public static class JudoHsqldbModulesBuilder {
+        private Boolean runServer = false;
+        private String databaseName = "judo";
+        private File databasePath = new File(".", "judo.db");
+        private Integer port = 31001;
+    }
+    @Builder
+    private JudoHsqldbModulesTest(Boolean runServer, String databaseName, File databasePath, Integer port) {
+        this.runServer = runServer;
+        this.databaseName = databaseName;
+        this.databasePath = databasePath;
+        this.port = port;
+    }
```

Comment:
I'm confused here. `@Builder` "should" also provide a builder class, why is `JudoHsqldbModulesBuilder` needed? If it's only for default, why not just check if values are null before assigning it since defaults are already defined in lines 44-47? hint: with [Objects#requireNonNullElse](https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/util/Objects.html#requireNonNullElse(T,T)) initial values in lines 44-47 are not needed.

Reply:
Removed added the original JudoHsqldbModules